@xen-orchestra/lite/src/components/Table.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Dictionary, isEmpty, map } from 'lodash'
+import { FormattedMessage } from 'react-intl'
+import { Map } from 'immutable'
+import { withState } from 'reaclette'
+
+import intlMessage from '../lang/en.json'
+
+type Collections = Dictionary<unknown> | unknown[]
+
+export type Column<Type> = {
+  formattedMessageId: keyof typeof intlMessage
+  propertyToDisplay: keyof Type
```

Comment:
We won't necessarily want to display a single property. We'll need something like a `render` option.